src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+
+interface Props {
+  hideErrorMessage: () => void,
+  errorMessage: string | null,
+}
+
+export const ErrorNotification: FC<Props> = (props) => {
+  const {
+    hideErrorMessage,
+    errorMessage,
+  } = props;
+
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className="notification is-danger is-light has-text-weight-normal"
```

Comment:
I think that error component should be rendered regardless of occurrence of any error, and if there isn't any, it should be hidden with class 'hidden' (as in README). Also, notification should be automatically hidden after 3 seconds

Reply:
done